algorithms/sorting/strand_sort.py

```diff
@@ -17,53 +17,46 @@
     Psuedo Code: https://en.wikipedia.org/wiki/Strand_sort
 """
 
-from collections import deque
 
-
-def sort(array):
-    if len(array) < 2:
-        return array
+def sort(unsorted):
+    if len(unsorted) < 2:
+        return unsorted
     result = []
-    while array:
-        sublist = [array.pop()]
+    while unsorted:
+        sublist = [unsorted.pop(0)]
+        leftovers = []
         last = sublist[0]
-        sub_append = sublist.append
-        leftovers = deque()
-        left_append = leftovers.append
-        for item in array:
+        sublist_append = sublist.append
+        leftovers_append = leftovers.append

```

Comment:
I think I would prefer them to be taken out. That way somebody doesn't go "Wait is `sublist_append` different from `leftovers_append`?", which is what I intially did.

As far as the other variables, I would prefer them to match what is in the other modules, but not a big deal right now. I will eventually change them up as I change the code to be more uniform.